modules/gmc-agent/src/main/java/org/gridgain/agent/Agent.java

```diff
@@ -96,7 +96,7 @@
     private NodeConfigurationService nodeConfigurationSrvc;
 
     /** Execute service. */
-    private ExecutorService execSrvc;
+    private ThreadPoolExecutor execSrvc;
```

Comment:
Lets rename `execSrvc` to `connectPool` ?

Reply:
Done.